app/models/bus.rb

```diff
@@ -0,0 +1,5 @@
+class Bus < ApplicationRecord
+  validates :bus_name, :bus_type, :bus_brand, :bus_capacity, presence:true
```

Comment:
what are the possible values of `bus_type`? Are they some fixed/previously defined values?

Reply:
Yes it is previously defined now as `['AC','NON_AC']`. Ans also changed the type into integer now and set the enum value as `enum typed: { AC: 0, NON_AC: 1 }`